package/com.unity.formats.usd/Runtime/Scripts/IO/SanitizedSamples.cs

```diff
@@ -17,6 +42,24 @@ public interface ISanitizable
         void Sanitize(Scene scene, SceneImportOptions importOptions);
     }
 
+    public class MeshConversionState : IConversionState
```

Comment:
If you rename `IConversionState` to `IPrimStreamingState`, I think this should also be `MeshStreamingState`